user/favorite/store.go

```diff
@@ -180,55 +117,38 @@ func (s *Store) FindAll(ctx context.Context, userID string, filter []assignment.
 		return nil, err
 	}
 
-	var allowServices, allowSchedules, allowRotations, allowEscalationPolicies, allowUsers bool
-	if len(filter) == 0 {
-		allowServices = true
-	} else {
-		for _, f := range filter {
-			switch f {
-			case assignment.TargetTypeService:
-				allowServices = true
-			case assignment.TargetTypeSchedule:
-				allowSchedules = true
-			case assignment.TargetTypeRotation:
-				allowRotations = true
-			case assignment.TargetTypeEscalationPolicy:
-				allowEscalationPolicies = true
-			case assignment.TargetTypeUser:
-				allowUsers = true
-			}
-		}
+	args := gadb.UserFavFindAllParams{
+		UserID:                  uuid.MustParse(userID),
+		AllowServices:           len(filter) == 0 || slices.Contains(filter, assignment.TargetTypeService),
+		AllowSchedules:          len(filter) == 0 || slices.Contains(filter, assignment.TargetTypeSchedule),
+		AllowRotations:          len(filter) == 0 || slices.Contains(filter, assignment.TargetTypeRotation),
+		AllowEscalationPolicies: len(filter) == 0 || slices.Contains(filter, assignment.TargetTypeEscalationPolicy),
+		AllowUsers:              len(filter) == 0 || slices.Contains(filter, assignment.TargetTypeUser),
```

Comment:
Do we want to set all of these to true is `len(filter) == 0`? It looks like before we were only doing so for `allowServices`, and the rest were dependent on check within the loop

Reply:
I believe that was a bug. If we don't specify a filter, all favorites should be returned. 

Originally only services supported favorites, and the code was just never updated when we added additional types.